src/main/resources/application.yaml

```diff
@@ -9,8 +9,11 @@ management:
     enabled: ${MANAGEMENT_SECURITY_ENABLED:true}
     allowedOrigin: ${ALLOWED_FRONTEND_ORIGIN:*}
     allowedHeaders: ${ALLOWED_HEADERS:*}
+    exposeHeaders: ${EXPOSE_HEADERS:Refreshed-Token}
```

Comment:
and where is that value EXPOSE_HEADERS coming from on Azure?

Reply:
nowhere, just as all the other values above it.
While you are asking about it I think it should be either static or taken from the vault and reused somehow in the frontend.